### DISCUSSION TITLE
Is this project still maintained?

### DISCUSSION THREAD

**Discussion by ZacSweers** [diffplug/spotless#2230] [Q&A] [answered]
It's been a few months since the last release, and we have a growing queue of missed ktfmt releases we can't use due to an ABI incompatibility introduced in a past release.

**Comment by nedtwigg** [comment, marked as answer]
Sorry for the lag, still maintained but not a top priority. Just published new release. I've laid out a more explicit SLA here:

- #2235

**Reply by ZacSweers** [reply]
Thanks!
